utils/log.go

```diff
@@ -0,0 +1,96 @@
+package utils
+
+import "fmt"
+
+func FormatWritesLog(codes []byte, line *string) {
+	n := len(codes)
+	str := ""
+	exist := false
+	if n == 3 {
+		if str, exist = ASCIIGroupToStr(fmt.Sprintf("%X", codes)); exist {
+			*line += str
+			codes = nil
+		}
+	}
+	// sh control codes
+	if n >= 6 {
+		if str, exist = ASCIIGroupToStr(fmt.Sprintf("%X", []byte{codes[0], codes[1], codes[n-1]})); exist {
+			*line += str
+			codes = nil
+		}
+	}
```

Comment:
I'm having trouble following this one. What does it mean for the first two bytes and the last byte to be `1B`, `5B`, and ` 52`, respectively? And why does that indicate that we don't want to log it?

Reply:
These characters are sent by termial after we type "return".  Used to indicate row and column numbers.  I'll add some more comments to illustrate.

Reply:
If possible, try this container image: `curlimages/curl`. Maybe, you can run like this:

```
./gotty -w docker run --rm -it curlimages/curl sh
```

Reply:
But if that's coming from the shell, why would we see it? Aren't you recording input?

Reply:
I'm sorry for my mistake. These control sequences are sent by the client(xterm.js). 
More information can be found here:
https://en.wikipedia.org/wiki/ANSI_escape_code
https://xtermjs.org/docs/api/vtfeatures/
But it should be noted that not every sequence we have to deal with.
For this part, I am optimizing the code. I will submit it to you for review as soon as possible. But I may not be able to handle all the control sequences